ts/parts-more/BoxPlotSeries.ts

```diff
@@ -723,4 +788,64 @@ seriesType<Highcharts.BoxPlotSeries>('boxplot', 'column', {
  * @apioption series.boxplot.data.q3
  */
 
+/**
+ * The dash style of the box.
+ *
+ * @sample {highcharts} highcharts/plotoptions/box-plot-styling/
+ *         Box plot styling
+ * @sample {highcharts} highcharts/css/boxplot/
+ *         Box plot in styled mode
+ *
+ * @type      {Highcharts.DashStyleValue}
+ * @default   Solid
+ * @since     next
+ * @product   highcharts
+ * @apioption series.boxplot.data.boxDashStyle
+ */
+
```

Comment:
Do need to define twice all options for docs?

Reply:
I think yes is we want to show them in API not only on the series level but on the point level too: https://api.highcharts.com/highcharts/series.boxplot.data
What is the better practice? Should I remove them from `boxplot.data` level?

Reply:
No, my bad, didn't realize it's `boxplot.data`, not `boxplot`. Thanks!